### ISSUE TITLE
More API's / SteamCalls for plugins

### ISSUE THREAD

**Issue by Trojaner25** [SmartlyDressedGames/Unturned-3.x-Community#44]
If I'm not wrong `askBrowserRequest` is the only SteamCall made for plugins at the moment.

We could need more:
For example

- one that changes client-side gravity
- more GUI  related stuff
etc

**Comment by Trojaner25** [comment]
Wow. Just, wow.

**Comment by zoomy500** [comment]
inb4 someone has to look for an arbitrary code execution exploit from server just to create clientside dynamic guis

**Comment by Trojaner25** [comment]
When GUI's can be defined with HTML etc @zoomy500 ...

But the main request isnt even GUI-related, just more freedom for plugin devs

**Comment by RoyIL** [comment]
LoL, you just refused? We're trying to make Unturned better but yeah.

**Comment by RoyIL** [comment]
At least leave a reason.

**Comment by aviadmini** [comment]
> important issue about steamcalls is closed
> NOPE GOTTA MAKE THAT EXPLOSIVE CHEWING GUM

>nelsonlogic

**Comment by fr34kyn01535** [comment]
If you guys have special ideas we can talk about it. It would be even better if you propose full implementation proposals.
Please stop both double posting and pointless comments on issues.

**Comment by snowball7275** [comment]
This isn't pointless. Adding explosive chewing gum is pointless. 
> "Special ideas"

Want a full implementation proposal? Make steamcalls that allow creation and manipulation of custom Sleek GUI classes, as well as steamcalls called when one of those custom GUI elements is clicked.

Want another one that would be nice for developers? The option to manipulate scene elements by name or other identifier through a steamcall.

Want another one? Allow manipulation of clientside player-related things, such as constant values in certain classes.

**Comment by Trojaner25** [comment]
> If you guys have special ideas we can talk about it. It would be even better if you propose full implementation proposals.
> Please stop both double posting and pointless comments on issues.

Nothing of this justifies the rudeness of Nelson by just closing a long-time-requested feature without any comment. Not even a single label. Just shows how much "worth" we plugin-developers are to Nelson.

**Comment by SDGNelson** [comment]
I closed the issue while cleaning up my to-do list for today because it is very broad/general. The steamcall you referenced (askBrowserRequest) was added for mod request so I'm definitely open to adding such features. My original intention with this issues board was simply to show what I'm working on - it would take a lot of time to reply to all the suggestions/requests posted everyone e.g. e-mail, Steam messages, forums, Steam discussions, discord, twitter.
